docs/sources/developers/plugins/share-a-plugin.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Share a plugin"
+type = "docs"
++++
+
+# Share a plugin
+
+You've just built your first plugin and now you want to share it with the world. In this guide, you'll learn how to package and share your plugin with others.
+
+When you build a plugin from source, a `dist` directory is created that contains the production build, or _plugin assets_, for your plugin.
+
+When loading your plugin, Grafana only cares about the plugin assets. Specifically, when the Grafana server starts, it attempts to discover and load plugins like this:
+
+1. Look for for a `plugin.json` file in any of the subdirectories in the plugin directory
```

Comment:
there's a duplicate "for" in this sentence

Reply:
Please put periods at the end of each sentence in this list.